### ISSUE TITLE
LanguageTool flags e-mail address at end of sentence

### ISSUE THREAD

**Issue by infotexture** [asciidoctor/asciidoctor-intellij-plugin#846]
## Observed vs. expected behavior

The [LanguageTool](https://languagetool.org) grammar & spell checker complains when an e-mail address is used at the end of a sentence.

## Steps to reproduce

For example, with the following sentence in a German document:

> Bei Fragen wenden Sie sich bitte an support@acme.com.

LanguageTool complains with the following message:

> “Vor dem Punkt sollte kein Leerzeichen stehen.”

There is no space before the final period in the sentence, which suggests that whatever logic is used to exclude e-mail addresses from spellchecking may lead LanguageTool to think there's a space before the period.

## Environment

Plugin Version: 0.33.18

### IntelliJ Details

```
IntelliJ IDEA 2021.2.1 (Community Edition)
Build #IC-212.5080.55, built on August 24, 2021
Runtime version: 11.0.11+9-b1504.16 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 11.5.2
GC: ParNew, ConcurrentMarkSweep
Memory: 1981M
Cores: 8
Registry: ide.balloon.shadow.size=0
Non-Bundled Plugins: org.asciidoctor.intellij.asciidoc (0.33.18)
Kotlin: 212-1.5.10-release-IJ5080.55
```

**Comment by ahus1** [comment]
Thanks for providing this example. This will be fixed in the next release. I'll update this issue once a pre-release is available.

**Comment by infotexture** [comment]
Brilliant, thanks for the quick fix! 🙏

**Comment by ahus1** [comment]
This is fixed in pre-release 0.34.1 of the plugin. Please give it a try.

The pre-release of the plugin is available from [GitHub releases](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/installing-a-preview-release.html) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).

**Comment by infotexture** [comment]
Thanks, works like a charm. 🙇 

I confirmed that the problem is indeed solved in 0.34.1. 

Thanks again for this plug-in, and for taking such good care of it. 🙏
